tscompare/methods.py

```diff
@@ -438,14 +469,15 @@ def f(t):
         shape=(ts.num_nodes, other.num_nodes),
     )
     time_discrepancies = np.asarray(
-        time_matrix[np.arange(len(best_match)), best_match].reshape(-1)
+        time_matrix[np.arange(len(best_n1_match)), best_n1_match].reshape(-1)
     )
     product = np.multiply((time_discrepancies**2), ts_node_spans)
     rmse = np.sqrt(np.sum(product) / total_span_ts)
     return ARFResult(
-        arf=1.0 - total_match_span / total_span_ts,
-        tpr=total_match_span / total_span_other,
-        dissimilarity=total_span_ts - total_match_span,
+        arf=1.0 - total_match_n1_span / total_span_ts,
+        tpr=total_match_n2_span / total_span_other,
+        dissimilarity=total_span_ts - total_match_n1_span,
+        inverse_dissimilarity=total_span_other - total_match_n2_span,
```

Comment:
proposal to change these to to
```suggestion
        matched_span = (total_match_n1_span, total_match_n2_span)
```